ipatests/test_xmlrpc/tracker/user_plugin.py

```diff
@@ -196,24 +196,26 @@ def track_create(self):
             ipantsecurityidentifier=[fuzzy_user_or_group_sid],
             )
 
-        for key in self.kwargs:
-            if key == u'krbprincipalname':
+        for key, value in self.kwargs.items():
+            if key == "krbprincipalname":
                 try:
-                    self.attrs[key] = [u'%s@%s' % (
-                        (self.kwargs[key].split('@'))[0].lower(),
-                        (self.kwargs[key].split('@'))[1]
-                    )]
+                    princ_splitted = value.split("@")
```

Comment:
Should this split include maxsplit=1?

Reply:
sure, thank you.

Reply:
Honestly I only saw it because of a previous change you made :-)